app/views/play/level/modal/CourseVictoryModal.coffee

```diff
@@ -32,6 +32,7 @@ module.exports = class CourseVictoryModal extends ModalView
 
     @session = options.session
     @level = options.level
+    @capstoneStage = Number(options.capstoneStage)
```

Comment:
What happens if the query param is not a recognizable number by JS and `NaN` is returned?

Is there a reason for using `Number` instead of `parseInt()`?

Reference: https://gomakethings.com/converting-strings-to-numbers-with-vanilla-javascript/

Reply:
In case NaN is returned (i.e. for any string with characters other than numbers), capstoneStage will be treated as undefined. I believe `parseInt` could still return a number for strings like `10px`, therefore `Number` seems to be better choice here since we always expect a numeric character here.

Also, I am glad you asked this question, I will update the logic to treat undefined capstone stage as stage 1 so that it returns the next level accordingly instead of a random next level.